synapse/storage/group_server.py

```diff
@@ -846,19 +846,25 @@ def _remove_user_from_group_txn(txn):
             )
         return self.runInteraction("remove_user_from_group", _remove_user_from_group_txn)
 
-    def add_room_to_group(self, group_id, room_id, is_public):
-        return self._simple_insert(
+    def update_room_group_association(self, group_id, room_id, is_public=True):
```

Comment:
looks like the default is never used? I'd be inclined to leave it unspecified so that the caller has to specify it

Reply:
WFM